artifactory_cleanup/rules/filters.py

```diff
@@ -68,3 +68,18 @@ class ExcludeFilename(FilterRule):
     attribute_name = "name"
     operator = "$nmatch"
     boolean_operator = "$and"
+
+
+class FilterByRegexpPath(Rule):
+    """
+    Filter artifacts paths by regex pattern.
+    """
+
+    def init(self, path):
+        self.path = rf"{path}"
+
+    def filter(self, artifacts: ArtifactsList) -> ArtifactsList:
```

Comment:
Could you import the `ArtifactsList` in the file so tests are passed? :pray:

Reply:
remember to run pre-commit after adding, it'll likely resort imports properly.
```python
pre-commit install
pre-commit run --all-files
```

Reply:
sorry forgot that while moving the block inbetween files.